how_to_solve_a_problem.md

```diff
@@ -0,0 +1,92 @@
+ # How to Solve a Problem
+
+Mastering problem solving will get you far. There are some techniques to accomplish this, and it's important to be aware of the fact that your day is mostly composed of problems that need to be solved.
+
+A dev who encounters a nasty bug; an account manager working on a price estimation; a salesperson trying to reach more leads. These are all big problems, which probably have solutions.
+
+As we will refer to "problem" as "bug" this chapter may seem only relevant for a developer. However, the methods and mindset applies to all the roles in the company.
```

Comment:
I prefer to tackle them in a follow up - mostly because I don't know how to answer this ;)